{
    "type": "vulnerability",
    "spec_version": "2.1",
    "id": "vulnerability--b54f56b6-2573-433e-bcb5-dacc5bf58524",
    "created": "2024-08-13T16:15:37.273163Z",
    "modified": "2024-08-13T16:15:37.273168Z",
    "name": "No Title Available",
    "description": "A vulnerability was discovered in vdsm, version 4.19 through 4.30.3 and 4.30.5 through 4.30.8. The systemd_run function exposed to the vdsm system user could be abused to execute arbitrary commands as root.",
    "external_references": [
        {
            "source_name": "cve",
            "external_id": "CVE-2019-3831"
        }
    ],
    "metrics": [
        {
            "cvssV3_0": {
                "attackComplexity": "HIGH",
                "attackVector": "LOCAL",
                "availabilityImpact": "HIGH",
                "baseScore": 6.4,
                "baseSeverity": "MEDIUM",
                "confidentialityImpact": "HIGH",
                "integrityImpact": "HIGH",
                "privilegesRequired": "HIGH",
                "scope": "UNCHANGED",
                "userInteraction": "NONE",
                "vectorString": "CVSS:3.0/AV:L/AC:H/PR:H/UI:N/S:U/C:H/I:H/A:H",
                "version": "3.0"
            }
        }
    ],
    "references": [
        {
            "tags": [
                "x_refsource_CONFIRM"
            ],
            "url": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2019-3831"
        }
    ]
}